Csrs

paigerube14 opened the pull request:
https://github.com/openshift-scale/cerberus/issues/96

Adding monitoring of csrs. I am logging a warning for any csr that does not have **Approved** as the status type. Not sure if there is a better way to do this. 

I did not add this to the items that are run in parallel. Not sure if I needed based on how quickly this could run and there possibly not being any csr's for most runs.

yashashreesuresh commented on start_cerberus.py:
```diff
@@ -328,6 +328,20 @@ def main(cfg):
 
+                if distribution == "openshift":
+                    watch_csrs_start_time = time.time()
+                    iter_track_time['watch_namespaces'] = time.time() - watch_csrs_start_time
```
Why is the time taken for watch_namespaces check being updated here?

paigerube14 replied:
Copied the wrong item. Took this out now

yashashreesuresh reviewed (commented):
Maybe we can move this check before custom_checks as we perform different checks and then report on slack and proceed inspect_components feature.
I think we can mention about csr's in README as well.

paigerube14 commented:
> Maybe we can move this check before custom_checks as we perform different checks and then report on slack and proceed inspect_components feature.
> I think we can mention about csr's in README as well.

Moved to above custom checks and added line to bottom of README

yashashreesuresh reviewed (approved):
LGTM
